pkg/cmd/cli/cmd/whoami.go

```diff
@@ -50,3 +54,39 @@ func (o WhoAmIOptions) WhoAmI() (*userapi.User, error) {
 
 	return me, err
 }
+
+func RunWhoAmI(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, args []string, o *WhoAmIOptions) error {
```

Comment:
I prefer the style of building an object that describes what should be done and then executing that separate from the cmd itself.